Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -1274,19 +1257,23 @@ def get_remote_data_command(service: client.Service, args: dict, close_incident,
             updated_notable.get("owner")) if mapper.should_map else updated_notable.get("owner")
 
     demisto.debug('notable {} data: {}'.format(notable_id, updated_notable))
-    if updated_notable.get('status') == '5' and close_incident:
-        demisto.info('Closing incident related to notable {}'.format(notable_id))
-        entries = [{
-            'Type': EntryType.NOTE,
-            'Contents': {
-                'dbotIncidentClose': True,
-                'closeReason': 'Notable event was closed on Splunk.'
-            },
-            'ContentsFormat': EntryFormat.JSON
-        }]
+    if close_incident and updated_notable.get('status_label') is not None:
```

Comment:
Why "is not None"? 
We want if `close_incident = True & updated_notable.get('status_label') == '' ` to get inside this code?

Reply:
This is just a safety check in case we receive an unexpected API response that doesn't include a 'status_label' key (which I don't believe should happen) to avoid a `KeyError` exception.

In such case, we just skip the mirror for the current specific Splunk Notable instead of crashing and thus avoiding mirror for other valid Notables returned by the API that are also being mirrored in the current run.

I'll add an `else` clause with a log saying the key couldn't be found for better debugging.

Reply:
```suggestion
    if close_incident and updated_notable.get('status_label', ""):
```
So why not like this?

Reply:
It has practically the same effect if a `status_label` is missing, so behaviorwise, it's the same.

It's also the same as:
``` python
updated_notable.get('status_label', {}):
```

and:
``` python
updated_notable.get('status_label', []):
```

I just don't see why alter the default `None` value returned by the `get()` function.

We check if a `status_label` key exists, and if it does not, returning the default `None` value seems more readable and by-convention to me than returning an empty string.

If you want to cover an edge case of a case where the key exists with an empty string value (which I don't think is needed), we can just do:
``` python
if close_incident and updated_notable.get('status_label'):
```

Reply:
When I proposed you before the `updated_notable.get('status_label', "")` is to show you that we don't need the check `if ___ is not None`.  
So let's keep it as simple as possible 
`if close_incident and updated_notable.get('status_label'):`

Reply:
My mistake, I misunderstood your first note.

```suggestion
    if close_incident and updated_notable.get('status_label'):
```